dist/templates/ovnkube-master-cluster-manager-nc-manager.yaml.j2

```diff
@@ -0,0 +1,354 @@
+# ovnkube-master
+# daemonset version 3
+# starts master daemons (ovnkube-cluster-manager, ovnkube-network-controller-manager and ovn-northd) each in a separate container
+# it is run on the master(s)
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-master
+  # namespace set up by install
+  namespace: ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This Deployment launches the ovn-kubernetes master networking components.
+spec:
+  progressDeadlineSeconds: 600
+  replicas: {{ ovn_master_count | default(1|int) }}
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      name: ovnkube-master
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: ovnkube-master
+        component: network
+        type: infra
+        kubernetes.io/os: "linux"
+    spec:
+      priorityClassName: "system-cluster-critical"
+      # Requires fairly broad permissions - ability to read all services and network functions as well
+      # as all pods.
+      serviceAccountName: ovn
+      hostNetwork: true
+      dnsPolicy: Default
+
+      # required to be scheduled on a linux node with node-role.kubernetes.io/control-plane label and
+      # only one instance of ovnkube-master pod per node
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: node-role.kubernetes.io/control-plane
+                    operator: In
+                    values:
+                      - ""
+                  - key: kubernetes.io/os
+                    operator: In
+                    values:
+                      - "linux"
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: name
+                    operator: In
+                    values:
+                      - ovnkube-master
+              topologyKey: kubernetes.io/hostname
+
+      containers:
+      # ovn-northd - v3
```

Comment:
why is northd within this pod? Its it just supposed to be ovnk master components (CM + NC mgrs)?

Reply:
The present ovnkube-master pod has ovn-northd too.

This yaml file will be used when CLUSTER_MANAGER_MODE is "inservice" which means we will have 
ovnkube-master pod with 3 containers
   - cluster manager
   -  network controller manager
   -  ovn-northd
We will also have ovnkube-db pod and ovnkube-node pod (as is the case  even now).

Reply:
ah ok - sorry for noise Numan. I got confused between downstream and upstream :)

Reply:
no worries